charts/fleet-crd/templates/crds.yaml

```diff
@@ -2103,6 +2103,9 @@ spec:
                 type: string
               forceSyncGeneration:
                 type: integer
+              helmRepoUrlRegex:
```

Comment:
I'd prefer helmRepoURLRegex to be consistent with the other spellings in that file

Reply:
done in https://github.com/rancher/fleet/pull/1234/commits/9434f279501df414114de8b0447bfb899db9c871